package.json

```diff
@@ -26,7 +26,8 @@
     "update-deps": "lerna exec '../../node_modules/.bin/ncu --upgrade' && lerna exec 'npm i --package-lock-only' && npm run reinstall-deps",
     "test": "lerna run test --stream",
     "test:debug": "lerna run test:debug --stream",
-    "lint": "node misc/is_eol || (tsc && lerna run lint --stream && eslint misc && eslint dockerfile-examples)",
+    "lint": "node misc/is_eol || (npm run typecheck && lerna run lint --stream && eslint misc && eslint dockerfile-examples)",
+    "typecheck": "tsc",
```

Comment:
This is an unrelated change I had lying around locally. The only thing it changes is that type checking via tsc is now available as a separate npm script.